app/src/main/java/com/framgia/musixmatch/ui/search/MySuggestionProvider.java

```diff
@@ -0,0 +1,17 @@
+package com.framgia.musixmatch.ui.search;
+
+import android.content.SearchRecentSuggestionsProvider;
+
+/**
+ * Created by superme198 on 20,March,2019
+ * in Music_Match_1.
+ */
+public class MySuggestionProvider extends SearchRecentSuggestionsProvider {
+
+    public final static String AUTHORITY = "com.framgia.musixmatch.ui.search.MySuggestionProvider";
```

Comment:
```suggestion
    public final static String AUTHORITY = "vn.sunasterisk.musixmatch.ui.search.MySuggestionProvider";
```
Em nên đổi tên https://stackoverflow.com/questions/16804093/rename-package-in-android-studio